lib/checks/keyboard/page-has-elm-after.js

```diff
@@ -0,0 +1,9 @@
+const elmUsedAnywhere = results.some(frameResult => frameResult.result === true);
+
+// If the element exists in any frame, set them all to true
+if (elmUsedAnywhere) {
+	results.forEach(result => {
+		result.result = elmUsedAnywhere;
```

Comment:
You're setting each frame's result to the value of `elmUsedAnywhere` when in truth, you only want to use this value when it's `true`. I think `result.result = true` would be more declarative.